models/api_host.go

```diff
@@ -89,7 +88,7 @@ func (a *ApiHost) ConvertAPIHostToNMHost(currentHost *Host) *Host {
 	h.IsK8S = currentHost.IsK8S
 	h.IsStatic = a.IsStatic
 	h.ListenPort = a.ListenPort
-	h.LocalListenPort = currentHost.ListenPort
+	h.ProxyListenPort = a.ProxyListenPort
```

Comment:
Will need to account for public port, correct? Otherwise it may get overwritten to 0

Reply:
there is no need to add public port to API model